netlify.toml

```diff
@@ -1,12 +1,32 @@
 # Settings in the [build] context are global and are applied to all contexts unless otherwise overridden by more specific contexts.
 # The cp, bundle install and bundle exec parts are only required to work around not being able to put the Gemfile int he root directory. It can't go in the root because that breaks the jekyll/jekyll container.
 [build]
-  command = 'export PATH=$PATH:/opt/build/repo/bin; cp netlify/Gemfile Gemfile; make bin/helm; bundle install && bundle exec jekyll build'
+  command = 'jekyll build'
+  publish = "_site/"
+
+[build.environment]
+  RUBY_VERSION = "2.5.4"
+
+# One jekyll build here generates the website which is served at /archive/<version> other build generates only manifests
+# served on /<version>/ path, order of the build matters since destination folders are cleaned on each builds
+# 404 page moved to root of the site so that netlify shows it automatically [context.branch-deploy]
+  command = '''
```

Comment:
just want to verify - is this `command block` correctly a child of `[build.environment]`?

Reply:
good catch, the `[context.branch-deploy]` was mistakenly got included in the comment. This still works because it's treated as child of `[build.environment]` now. I'll fix this.